### PR TITLE
docs(ReactUI): Updates for meeting 1

### PR DESCRIPTION
**Pull request by krishna9304**
## Project

React UI

### Changes

Community Bonding Meeting 1 updates added.

### CODE REVIEW

File: docs/2022/ui/updates/krishna/2022-06-02.md [hunk 1]
```diff
@@ -12,21 +12,27 @@ SPDX-License-Identifier: CC-BY-SA-4.0
 SPDX-FileCopyrightText: 2022 Krishna Mahato <@gmail.com>
```
Comment (GMishx) [inline docs/2022/ui/updates/krishna/2022-06-02.md:12]: Please fix the email in copyright.

File: docs/2022/ui/index.md [hunk 2]
```diff
@@ -3,14 +3,14 @@ sidebar_position: 1
...
 ---
+
 <!--
 SPDX-License-Identifier: CC-BY-SA-4.0
 
 SPDX-FileCopyrightText: 2022 Gaurav Mishra <mishra.gaurav@siemens.com>
 SPDX-FileCopyrightTest: 2022 Siemens AG
 -->
 
-
```
Comment (GMishx) [inline docs/2022/ui/index.md:13]: Please revert unnecessary changes.

File: docs/2022/ui/index.md [hunk 3]
```diff
@@ -3,14 +3,14 @@ sidebar_position: 1
 title: Introduction
 slug: /2022/ui/
 ---
+
```
Comment (GMishx) [inline docs/2022/ui/index.md:6]: Please revert unnecessary changes.

**Review by GMishx — CHANGES_REQUESTED** [review]

File: docs/2022/ui/updates/krishna/2022-06-02.md (hunk 1 above)
Comment (krishna9304) [reply docs/2022/ui/updates/krishna/2022-06-02.md]: Done!

File: docs/2022/ui/index.md (hunk 2 above)
Comment (krishna9304) [reply docs/2022/ui/index.md]: Sure, Done.

**Review by GMishx — APPROVED** [review]

**Pull request merged**